.buildkite/dagster-buildkite/dagster_buildkite/steps/integration.py

```diff
@@ -32,7 +32,7 @@ def build_integration_steps() -> List[BuildkiteStep]:
     # test suites
     steps += build_backcompat_suite_steps()
     steps += build_celery_k8s_suite_steps()
-    steps += build_k8s_suite_steps()
+    # steps += build_k8s_suite_steps()
```

Comment:
back link to PR with listed failures https://github.com/dagster-io/dagster/pull/16605

current hypothesis is that this is related to https://health.aws.amazon.com/health/home#/account/dashboard/open-issues?eventID=arn:aws:health:us-west-2::event/MULTIPLE_SERVICES/AWS_MULTIPLE_SERVICES_OPERATIONAL_ISSUE/AWS_MULTIPLE_SERVICES_OPERATIONAL_ISSUE_F58A8_CCE53E4008A&eventTab=details